### ISSUE TITLE
It would be nice and logical if these buttons are also in the gas-load page

### ISSUE THREAD

**Issue by ChaelKruip** [quintel/etmoses#925]
<img width="573" alt="screen shot 2016-04-19 at 14 19 19" src="https://cloud.githubusercontent.com/assets/1303760/14638865/ce982ef2-0639-11e6-8ee1-4de3e22d4356.png">

**Comment by antw** [comment]
Something like this would be nice. Would probably be a fair bit of work to get the gas charts to reload when changing strategies, but it would be cool:

![screen shot 2016-04-20 at 17 09 25](https://cloud.githubusercontent.com/assets/4383/14681823/aaab1144-071a-11e6-9c20-42cf84c5042e.png)

**Comment by ChaelKruip** [comment]
> Would probably be a fair bit of work to get the gas charts to reload when changing strategies

Wouldn't it be just as simple/difficult as with the electricity load chart?

**Comment by grdw** [comment]
> Wouldn't it be just as simple/difficult as with the electricity load chart?

No, the whole technical background is entirely different. Also what would be the expected outcome in the gas chart; multiple charts with strategies enabled and disabled? The thing is that that would take a considerable amount of time like @antw said.
